framework/include/media/MediaTypes.h

```diff
@@ -1,77 +1,76 @@
-/* ****************************************************************

```

Comment:
Is it okay to submit this kind of change?
Please check this with @sunghan-chang and need to get a confirmation from him.